utils/sendVerificationRequest.js

```diff
@@ -0,0 +1,123 @@
+import nodemailer from 'nodemailer'; // Nodemailer to send email
+
+/**
+ * Sends verification request email via nodemailer
+ * @param {String} email of requesting user
+ * @param {String} url to use to authenticate user
+ * @param {Object} provider to feed information about mail specifics
+ * @returns promise, resolving to a successful nodemailer email send
+ */
+const sendVerificationRequest = ({ identifier: email, url, provider }) => {
+  // Return a new promise
+  return new Promise((resolve, reject) => {
+    // Collect mail server and from email
+    const { server, from } = provider;
+
+    // Create mail with server
+    nodemailer.createTransport(server).sendMail(
```

Comment:
This code seems to be the same as the code in aws/index.js. Perhaps you could just import that file instead?